package.json

```diff
@@ -8,6 +8,7 @@
     "start": "bsb -make-world -w",
     "coverage": "nyc report --temp-directory=coverage --reporter=text-lcov | coveralls",
     "test": "npm run build && jest --coverage && npm run coverage && npm run test:examples",
+    "test:nocoverage": "npm run build && jest && npm run test:examples",
```

Comment:
this can be removed; but the original `test` script was giving me errors when trying to run coverage, relating to the git repo